source/includes/_me.md

```diff
@@ -411,6 +411,8 @@ Parameter | Type | Default | Description
 `isFinished` | Boolean | `false` | Whether the media is finished.
 `hideFromContinueListening` | Boolean | `false` | Whether the media will be hidden from the "Continue Listening" shelf.
 `lastUpdate` | Integer | `Date.now()` | The time (in ms since POSIX epoch) when the media progress was last updated.
+`finishedAt` | Integer | `Date.now()` | The time (in ms since POSIX epoch) when the user finished the media.
+`startedAt` | Integer | `finishedAt` | The time (in ms since POSIX epoch) when the user started consuming the media.
```

Comment:
`startedAt` doesn't do anything even after that update you made. Did you intend on using startedAt?

Reply:
Damn, seems like I missed something.
Running a few tests it looks like it does not work properly if no progress was set before.
It does work if one was already set:

```shell
❯ curl -s 'http://127.0.0.1:3333/api/me/progress/li_ywupqxw5d22adcadpa' -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJyb290IiwidXNlcm5hbWUiOiJyb290IiwiaWF0IjoxNjY4OTYxNjAxfQ._NbilCoFy_hfoqy7uvbV4E_0X6qgLYapQ_MoRptP0oI'                                                                                                                                       
Not Found

❯ curl 'http://127.0.0.1:3333/api/me/progress/li_ywupqxw5d22adcadpa' -X PATCH -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJyb290IiwidXNlcm5hbWUiOiJyb290IiwiaWF0IjoxNjY4OTYxNjAxfQ._NbilCoFy_hfoqy7uvbV4E_0X6qgLYapQ_MoRptP0oI' -H 'Content-Type: application/json' --data-raw '{"isFinished":true, "finishedAt": 1669158206000, "startedAt": 1668056852000}'
OK

❯ curl -s 'http://127.0.0.1:3333/api/me/progress/li_ywupqxw5d22adcadpa' -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJyb290IiwidXNlcm5hbWUiOiJyb290IiwiaWF0IjoxNjY4OTYxNjAxfQ._NbilCoFy_hfoqy7uvbV4E_0X6qgLYapQ_MoRptP0oI' | jq .startedAt                                                                                                                       
1669158206000   ← Did not work

# ↓ run the same request again
❯ curl 'http://127.0.0.1:3333/api/me/progress/li_ywupqxw5d22adcadpa' -X PATCH -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJyb290IiwidXNlcm5hbWUiOiJyb290IiwiaWF0IjoxNjY4OTYxNjAxfQ._NbilCoFy_hfoqy7uvbV4E_0X6qgLYapQ_MoRptP0oI' -H 'Content-Type: application/json' --data-raw '{"isFinished":true, "finishedAt": 1669158206000, "startedAt": 1668056852000}'
OK

❯ curl -s 'http://127.0.0.1:3333/api/me/progress/li_ywupqxw5d22adcadpa' -H 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySWQiOiJyb290IiwidXNlcm5hbWUiOiJyb290IiwiaWF0IjoxNjY4OTYxNjAxfQ._NbilCoFy_hfoqy7uvbV4E_0X6qgLYapQ_MoRptP0oI' | jq .startedAt
1668056852000   ← now it did work
```

I'll take a look and fix it.

Reply:
Found the problem. Sorry, that was a simple fix and a fairly stupid mistake:
https://github.com/advplyr/audiobookshelf/pull/1211

Reply:
Thanks, that works